<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<title>Jennifer Listgarten</title>

<meta name="viewport" content="width=device-width"/>
<meta name="description" content="The Curriculum Vitae of Joe Bloggs."/>
<meta charset="UTF-8">

<link type="text/css" rel="stylesheet" href="style.css">
<link href='http://fonts.googleapis.com/css?family=Rokkitt:400,700|Lato:400,300' rel='stylesheet' type='text/css'>

</head>

<body id="top">
<div id="cv">
	<div class="mainDetails">
		<div id="headshot">
			<img src="./Pictures/jenn.png"  alt="Jennifer Listgarten" style="width: 120%;" />
		</div>

		<div id="name">
			<h1>Jennifer Listgarten</h1>
		</div>

		<div id="contactDetails">
			<ul>
			<BR>

				<li>Mailing address:<BR>
				    University of California, Berkeley<BR>
					Electrical Engineering and Computer Science<BR>
					387 Soda Hall, MC 1776<BR>
					Berkeley, CA, 94720<BR>
				</li>
				<BR>
				<li>Before contacting me,
				<!---
				</li>
                 <li>CS 189/289 students read<A HREF="#cs189"><U> this</U></A>,</LI>
                 <li> other students read <A HREF="#students"><U>this</U></A>.</LI>
                 --->
                 students please read <A HREF="group.html#students"><U>this</U></A>.</LI>
                 <LI>jennl [at] berkeley.edu</LI>
			</ul>
		</div>

		<div class="clear"></div>
	</div>

<!------------------------------------->
	<div id="mainArea">
		<section>
			<article>
				<div class="sectionTitle">
					<h1>About me</h1>
				</div>

				<div class="sectionContent">
					<p>

					I am a Professor in UC Berkeley's
												<A HREF="https://eecs.berkeley.edu/">
												EECS department
												</A>,

												<A HREF="http://ccb.berkeley.edu">
												Center for Computational Biology</A>,

												<A href="https://bioegrad.berkeley.edu/">Bioengineering</A>,

					and a member of the steering committee for the
					<A href="http://bair.berkeley.edu/">Berkeley AI Research (BAIR) Lab</A>.


					From 2007 to 2017 I was at Microsoft Research, through
					<A HREF="http://research.microsoft.com/en-us/labs/newengland/">
					Cambridge, MA</A>, Los Angeles and
					<A HREF="https://www.microsoft.com/en-us/research/lab/microsoft-research-redmond/">
					Redmond, WA</A>. Before that I did my PhD in the machine learning group at the
					<A HREF="http://www.cs.toronto.edu:40292/">University of Toronto</A>.

 					(Feel free to use this
					<A href="bio.txt">short bio</A> and
					<A href="./Pictures/Jennifer Listgarten_highres_square.jpg">picture</A> for announcements.)

					<BR><BR>

					<U>My expertise is in machine learning, applied statistics, computational biology.</U> I'm interested in both
					methods development as well as application of methods to enable new insight into basic biology and medicine.
					Recently I have also started to work on computational chemistry.


					<!----
					My current focus has
					been on problems in statistical genetics, and in particular on association studies (genetic and epigenetic) in the presence of unknown confounders. I have also
					worked on problems in immunoinformatics, liquid-chromatography proteomics, and microarray analysis.
					--->

									A recent print interview focused on my CRISPR work can be found <A href="https://www.weareworldquant.com/en/thought-leadership/machine-learning-meets-genome-editing/">here</A>.


									If you're interested more generally in how machine learning and biology go together, check out this
									<A href="https://soundcloud.com/harvard/machine-learning-in-biology?in=harvard/sets/talking-machines">
									Talking Machines interview with me
									</A>
									instead
									(alternate link to audio <A href="./files/talkingMachines.mp3">here</A>).
									Finally, if you want to hear about my random walk in education & career space, take a look at this Berkeley Science Review
									<A HREF="http://berkeleysciencereview.com/article/faculty-profile/">profile</A>.

					<BR><BR>

					<U>Current areas of interest include</U>: computational methods for protein design/optimization/engineering for properties such as expression, flurorescence, binding, stability, etc.; related methods for molecule design and chemical reactions; drug repositioning and discovery; machine learning methods development, and in particular at the intersection of graphical models, neural networks and variational inference, as well as optimizing black box probablistic functions with pathological regions that can't be trusted.

					<BR><BR>

					<U>Previous focus areas include</U>: machine learning methods for time series alignment and normalization; LC-MS proteomics; statistical genetics methods to correct for confounding factors in GWAS, epigenome-WAS and eQTL studies; problems in immunoinformatics such as HLA class I epitope prediction and HLA allele imputation. If you're interested in my statistical genetics work (FaST-LMM or EWASher), please go to this
									<!---
									<A href="http://research.microsoft.com/en-us/um/people/jennl/fastlmm.htm">
									<A href="http://research.microsoft.com/en-us/projects/fastlmm">
									<A href="http://www.jennifer.listgarten.com/fastlmm.html">
									--->
									<A href="https://www.microsoft.com/en-us/research/project/fast-lmm-factored-spectrally-transformed-linear-mixed-models">
									landing page</a>.
					<BR><BR>

					<U>Industry engagements</U>:
					I also spend some of my time with companies. At the moment, these:
					 <LI>  <A href="https://www.dayzerodiagnostics.com/">Dayzero Diagnostics</a> (Scientific Advisory Board).
					 <LI>  <A href="https://deepappletx.com/">Deep Apple Therapeutics</a> (Scientific Advisory Board).
					 <!-----
					 <LI>  <A href="https://medium.com/@vikbajaj/foresite-labs-supporting-the-next-generation-of-healthcare-entrepreneurs-93a1dba2222e">Foresite Labs</a> (Scientific Advisory Board).
					 ------->
					 <LI>  <A href="https://www.inscripta.com/">Inscripta</a> (Scientific Advisory Board).
					 <LI>  <A href="https://www.fabletherapeutics.com"> Fable Therapeutics</A> (Academic co-founder).
					 <!-----
					 <LI>  <A href="https://patch.bio/">Patch Biosciences</a> (Scientific Advisory Board).
					 ------->

					 </UL>

<!-----
					<BR><BR>

					<U>Consulting</U>: Please direct consulting requests to my email address and use the word 'consulting' in the subject.
---->

					</p>
				</div>
			</article>
			<div class="clear"></div>
		</section>

		<!------------------------------------->
				<section>
					<div class="sectionTitle">
						<h1>Research group</h1>
					</div>

					<div class="sectionContent">

					<article>

						<p >
						You can find us
						<A HREF="group.html">here</A>.
						</p>

						<P>  If you are looking to do your PhD, applications go through academic entities such as
												<A HREF="https://eecs.berkeley.edu/academics/graduate/research-programs/admissions">EECS</A>
												the <A HREF="https://grad.berkeley.edu/program/computational-biology/">Center for
												Computational Biology</A>, or the <A href="https://bioegrad.berkeley.edu/">Bioengineering</A> program,
												for which you should list me as a faculty of interest.
												For EECS applicants, please choose as your primary area,
												<A href="https://ai-science.eecs.berkeley.edu/">CS:AI-SCIENCE</A>.
						</P>


					</div>
					<div class="clear"></div>
				</section>



<!------------------------------------->
		<section>
			<div class="sectionTitle">
				<h1>Announcements</h1>
			</div>

			<div class="sectionContent">

			<article>

<!----
				<h2><A name="cs189">CS 189/289 (Fall 2019)</A></h2>
				 <BR>
				 Due to the demand and class size I cannot answer emails about course enrollment, pre-requisites,
				 or otherwise. As noted on the class <A HREF="https://www.eecs189.org/">website</A>, all contact for the
				 class should first go through Piazza. Questions about enrollment should be directed to cs-enrollments@berkeley.edu,
				 or drop in on Cindy Conners' Drop-In Office Hours (August 28 - September 20: Monday-Friday between 1:00-4:00pm).
				 Note that the class is heavily over-subscribed and that there are no instructor-allocated spots,
				 therefore I cannot give you an enrollment code or somehow get you in.
				 Concurrent enrollment will not be allowed.
				 				 <BR><BR>
----->

<!------------------------------------------------------------------------------->

				<h2>New pieces</A></h2>
<BR>

				<article>
					 <H4>Guide your favorite protein sequence generative model</H4>
					 <p style="margin-left: 40px">
					Junhao Xiong*, Hunter Nisonoff*, Maria Lukarska*, Ishan Gaur, Luke M. Oltrogge, David F. Savage+, Jennifer Listgarten+
					 <BR>
					  <EM> arXiv</EM> 2025
					 <A HREF="https://arxiv.org/abs/2505.04823">(link</A>)
					 <BR>
						*(equal contributions), +(co-corresponding)
   			   </article>

				<article>
					 <H4>Unlocking Guidance for Discrete State-Space Diffusion and Flow Models</H4>
					 <p style="margin-left: 40px">
					{Hunter Nisonoff*, Junhao Xiong*, Stephan Allenspach*}, Jennifer Listgarten
					 <BR>
					  <EM> ICLR</EM> 2025
					 <A HREF="https://iclr.cc/virtual/2025/poster/29286">(link</A>)
						*(equal contributions)
					 <BR>
   			   </article>


			   <article>
					 <H4>Generative models for protein structures and sequences</H4>
					 <p style="margin-left: 40px">
					Chloe Hsu, Clara Fannjiang, Jennifer Listgarten
					 <BR>
					  <EM> Nature Biotechnology</EM> 2024
					 <A HREF="https://www.nature.com/articles/s41587-023-02115-w">(link</A>)
					 and a freely available copy <A HREF="./papers/s41587-023-02115-w.pdf">here</A>.

					 <BR>
   			   </article>

			   <article>
					 <H4>The perpetual motion machine of AI-generated data and the distraction of "ChatGPT as scientist"</H4>
					 <p style="margin-left: 40px">
					Jennifer Listgarten
					 <BR>
					  <EM> Nature Biotechnology</EM> 2024
					 <A HREF="https://www.nature.com/articles/s41587-023-02103-0?">(link</A>)
					 and a <A HREF="https://rdcu.be/dwSSy"> freely available version here</A>,
					 and a <A HREF="https://youtu.be/G43Em6ELaiE?si=Z7NTphG2C1yQS2Js&t=4130"> talk </A>
					 at the 2024 National Academy of Sciences annual meeting.

					 <BR>
   			   </article>





			<h2><A name="cs189">CS 294-150 Machine Learning and Statistics Meet Biology (Spring 2023)</A></h2>
							 <BR>
							 If you have questions about enrolling in this class, or otherwise, please read the
							  <A href="https://bcourses.berkeley.edu/courses/1541492">bcourses web page</A>, noting the year in case it's out of date. This site also contains a link for those unable to automatically enroll to fill out an application to enroll---please make sure only to fill out for the correct year (and please email me if the semester is about to start and it hasn't been updated).
							 <!----
							 read this, which I will update as the course approaches, including posting a zoom link in case you are not yet enrolled.
							 <A href="https://docs.google.com/document/d/1KHdjA72x5khK-EHAganEGeqxs2V4Fvj4pyVPEQKF5vM/edit">this</A>.
							 ---->
			</article>
			<BR>

			<article>

						<h2><A name="students">Undergraduate, graduate and post-doc opportunities</A></h2>

						<p>
						Please see "Joining the group" <A HREF="group.html#join">here</A>.
						</p>

		    </article>


			</div>
			<div class="clear"></div>
		</section>


<!------------------------------------->

		<section>
			<div class="sectionTitle">
				<h1>Selected<BR>Publications</h1>
			</div>

			<div class="sectionContent">


			<article><P>
				 <H3>Guide your favorite protein sequence generative model</H3>
				Junhao Xiong*, Hunter Nisonoff*, Maria Lukarska*, Ishan Gaur, Luke M. Oltrogge, David F. Savage+, Jennifer Listgarten+
				 <BR>
				  <EM> arXiv</EM> 2025
				 <A HREF="https://arxiv.org/abs/2505.04823">(link</A>)
				 <BR>
					*(equal contributions), +(co-corresponding)
		   </P></article>

			<article><P>
				 <H3>Unlocking Guidance for Discrete State-Space Diffusion and Flow Models</H3>
				{Hunter Nisonoff*, Junhao Xiong*, Stephan Allenspach*}, Jennifer Listgarten
				 <BR>
				  <EM> ICLR </EM> 2025
				 <A HREF="https://iclr.cc/virtual/2025/poster/29286">(link</A>)
					*(equal contributions)
				 <BR>
		   </P></article>


		   <article><P>
			<H3>The perpetual motion machine of AI-generated data and the distraction of "ChatGPT as scientist"</H3>
					Jennifer Listgarten
					 <BR>
					  <EM> Nature Biotechnology</EM> 2024<A HREF="https://www.nature.com/articles/s41587-023-02103-0?">(link</A>)
					 and a <A HREF="https://rdcu.be/dwSSy"> freely available version here</A>,
					 and a <A HREF="https://youtu.be/G43Em6ELaiE?si=Z7NTphG2C1yQS2Js&t=4130"> talk </A>
					 at the 2024 National Academy of Sciences annual meeting.
					 <BR>
   		   </P></article>

   		   <article><P>
		   				 <H3>Optimal trade-off control in machine learning-based library design, with application to adeno-associated virus (AAV) for gene therapy</H3>
		   				D Zhu*, DH Brookes*, A Busia*, A Carneiro, C Fannjiang, G Popova, D Shin, KC Donohue,, LF Lin, ZM Miller, ER Williams, EF Chang, TJ Nowakowski, <U>J Listgarten*</U>,  <U>DV Schaffer*</U>
		   				(*equal contributions, <U>corresponding</U>)
		   				 <BR>
		   				  <EM>Science Advances</EM> 2024
		   				 (<A HREF="https://www.science.org/doi/10.1126/sciadv.adj3786">paper link</A>)
		   				 <BR>
		   	</p></article>


		   <article><P>
				 <H3>Is novelty predictable?</H3>
				Clara Fannjiang and Jennifer Listgarten
				 <BR>
					<EM>Cold Spring Harbor Perspectives in Biology</EM> 2023
									 (<A HREF="https://doi.org/10.1101/cshperspect.a041469">paper link</A>),
				  similar version also available on <A HREF="https://arxiv.org/abs/2306.00872"> arXiv</A>).
				 <BR>
			</P></article>

			 <article><p>
				 <H3>MBE: model-based enrichment estimation and prediction for differential sequencing data</H3>
				Akosua Busia and Jennifer Listgarten
				 <BR>
				  <EM>Genome Research</EM> 2023 								 &nbsp;
				 <A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03058-w">(paper link</A>)
				 <BR>
			</p></article>


			<article>
					 <H3>Learning protein fitness models from evolutionary and assay-labeled data</H3>
					Chloe Hsu,  Hunter Nisonoff,  Clara Fannjiang, Jennifer Listgarten
					 <BR>
					  <EM>Nature Biotechnology</EM> 2022 								 &nbsp;
					 <A HREF="https://www.nature.com/articles/s41587-021-01146-5">(paper</A>)
					 <BR>
			</p></article>


			<article><p>
				 <H3>Rethinking drug design in the artificial intelligence era</H3>

				 P Schneider, WP Walters, AT Plowright, N Sieroka,
				 J Listgarten, RA Goodnow Jr., J Fisher,
				 JM Jansen, JS Duca, TS Rush, M Zentgraf,
				 JE Hill, E Krutoholow, M Kohler, J Blaney,
				 K Funatsu, C Luebkemann and G Schneider
				 <BR>
				 in <EM>Nature Reviews Drug Discovery</EM>
				 &nbsp;2019
				 <A HREF="https://www.nature.com/articles/s41573-019-0050-3?draft=collection">(paper link)</A>
				<BR>
			</p></article>

				<article><p>

			 <H3>Conditioning by adaptive sampling for robust design</H3>

			 David H. Brookes, Hahnbeom Park and Jennifer Listgarten

			 <BR>
			 accepted at <EM>ICML</EM>
			 &nbsp;2019
			  <A HREF="https://arxiv.org/abs/1901.10060">(paper link</A>, <I>arXiv</I> version is most up-to-date)
			 <BR>
			 (5% acceptance rate for 20 min. oral presentation)
			 <BR>
				</p></article>


				<article><p>
					<H3>
					 Predicting off-target effects for end-to-end CRISPR guide design
						</H3>
					 <U>J Listgarten</U>, <U>M Weinstein</U>, B Kleinstiver, AA Sousa, JK Joung, J Crawford, K Gao, M Elibol, L Hoang, <U>J Doench</U>, <U>N Fusi</U>
					 <U>(equal contributions and corresponding</U>)
					 <BR>
					 <EM>Nature Biomedical Engineering</EM>
					 &nbsp;(2018)
					 <A HREF="https://www.nature.com/articles/s41551-017-0178-6">(paper link)</A>
					 <BR>
					 <FONT color="#000099">
					   Associated tools and resources available
					  <A HREF="https://www.microsoft.com/en-us/research/project/crispr/">here</A>.
					 </FONT>
				</p></article>

				<article><p>
										<H3>
								 Orthologous CRISPR-Cas9 for Combinatorial Genetic Screens
									</H3>
								 F Najm*, C Strand*, K Donovan*, M Hegde*, KR. Sanson*, EW Vaimberg, ME Sullender, E Hartenian, N Fusi,
								 J. Listgarten, ST Younger*, BE Bernstein**, DE Root**, JG Doench**
								 <BR>
								 <EM>Nature Biotechnology</EM>
								 &nbsp;(2018)
								 <A HREF="https://www.nature.com/articles/nbt.4048">(paper link)</A>
								 (*equal contributions, **corresponding)
								 <BR>
				</p></article>


				<article>
									<p>
										<h3>
										Optimized sgRNA design to maximize activity and minimize off-target effects for genetic screens with CRISPR-Cas9
										<!--
										<img src="images/new.gif" >
										-->
										</h3>
										<U>JG Doench</U>*, <U>N Fusi</U>*, M Sullender*, M Hegde*, EW	Vaimberg*, KF Donovan, I Smith, Z Tothova,
										C Wilen , R Orchard , HW Virgin, <U>J Listgarten</U>*, <U>DE Root</U>
										<BR>
										<EM>Nature Biotechnology</EM>
										&nbsp; 2016
										<A HREF="http://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.3437.html">doi:10.1038/nbt.3437</A>
										<BR>
										(*equal contributions, <U>corresponding</U>)
										<BR>
										<FONT color="#000099">
										A pre-print of just the computational aspects of this paper is available
										on <EM> <A HREF="http://biorxiv.org/content/early/2015/06/26/021568">
												bioRxiv</A></EM>

										<BR>
										Source code and prediction server available from here:
										<A HREF="
										http://research.microsoft.com/en-us/projects/azimuth
										">here</A>.
										<BR>
										[<A HREF="http://blogs.microsoft.com/next/2016/01/18/molecular-biology-meets-computer-science-tools-in-new-system-for-crispr/">Microsoft Research blog post</A>]
										<BR>
										[<A HREF="https://www.broadinstitute.org/blog/machine-learning-approach-improves-crispr-cas9-guide-pairing">Broad Institute blog post</A>]

										</FONT>
								</p>
				<article>



	<article><p>
				<H3>
				 Epigenome-wide association studies without the need for cell-type composition
					</H3>
				 James Zou, C. Lippert, D. Heckerman, Martin Aryee, Jennifer Listgarten
				 <BR>
				 <EM>Nature Methods</EM>
				 &nbsp; 2014
				 <A HREF="http://www.nature.com/nmeth/journal/v11/n3/full/nmeth.2815.html"> (journal link)</A>
				 <BR>
				 <FONT color="#000099">
				  Python software available from <A HREF="http://research.microsoft.com/en-us/downloads/8af2ab12-a205-4bbb-809c-a333ecaffa40/">here</A>, and
				 R software available from <A HREF="http://research.microsoft.com/en-us/downloads/472fe637-7cb9-47d4-a0df-37118760ccd1/">here</A>.
				 </FONT>
				 <BR> Corrigendum: Supp. Figure 1 was not run with filters as described in the paper, but without any filters. We have contacted the journal to post this correction.
	</p></article>

	<article><p>
				<H3>
				FaST-LMM-Select for addressing confounding from spatial structure and rare variants
					</H3>
				<U>Jennifer Listgarten</U>, <U>Christoph Lippert</U>, <U>David Heckerman</U>
				(<U>equal contributions and corresponding</U>)
				<BR>
				<EM>
				Nature Genetics
				</EM>
				&nbsp; 2013
				(<A HREF="
				http://www.nature.com/ng/journal/v45/n5/full/ng.2620.html
				">journal link</A>)
			</p></article>


				<article><p>
				<H3>
				Improved linear mixed models for genome-wide association studies
					</H3>
				<U>Jennifer Listgarten</U>, <U>C. Lippert</U>, C. Kadie, R. Davidson, E. Eskin and <U>D. Heckerman</U>
				<BR>
				(<U>equal contributions and corresponding</U>)
				<BR>
				<EM>
				Nature Methods
				</EM>
				&nbsp; 2012,
				<A HREF="
				http://www.nature.com/nmeth/journal/v9/n6/abs/nmeth.2037.html
				">doi:10.1038/nmeth.2037
				</A>
				<BR>

				<FONT color="#000099">
				Source and executables available
				<A HREF="
				http://www.jennifer.listgarten.com/fastlmm.html
				">here</A>.
				</FONT>
				</p></article>


		<article><p>
				<H3>
				FaST Linear Mixed Models for Genome-Wide Association Studies
					</H3>
				<U>C. Lippert</U>, <U>Jennifer Listgarten</U>, Y. Liu, C.Kadie, R. Davidson and <U>D. Heckerman</U>
				<BR>
				(<U>equal contributions and corresponding</U>)
				<BR>
				<EM>
				Nature Methods
				</EM>
				&nbsp; Aug. 2011,
				<A HREF="
				https://www.nature.com/articles/nmeth.1681
				">doi:10.1038/nmeth.1681
				</A>,
				<BR>
				<A HREF="
				./papers/FaST-LMM-2011.pdf">
				freely available author copy including SI</A>,

				<FONT color="#000099">
				Source and executables available
				<A HREF="
				http://www.jennifer.listgarten.com/fastlmm.html
				">here</A>.
				</FONT>
		</p></article>

		<article><p>
				<H3>
				Correction for Hidden Confounders in the Genetic Analysis of Gene Expression
					</H3>
							Jennifer Listgarten, Carl Kadie, Eric Schadt, David Heckerman
				<BR>
				<EM>
				PNAS
				</EM>
				&nbsp; 2010, <A HREF="http://www.pnas.org/content/early/2010/08/30/1002425107.abstract">doi: 10.1073/pnas.1002425107
				</A>
				<BR>
				(<A
				HREF="http://www.pnas.org/content/early/2010/08/30/1002425107.full.pdf+html
				">paper</A>,
				<A
				HREF="http://www.pnas.org/content/suppl/2010/09/01/1002425107.DCSupplemental/Appendix.pdf
				">Supplementary Information</A>,
				<A
				HREF="http://mscompbio.codeplex.com/wikipage?title=eLMM&referringTitle=Home
				">software</A>,
				<A
				HREF="http://www.genomeweb.com/informatics/microsoft-research-led-team-develops-method-correct-confounders-eqtl-analyses
				">GenomeWeb coverage</A>)
		</p></article>



		 <article><p>

						<H3>

						Bayesian detection of infrequent differences in
						sets of time series with shared structure.
						</H3>
						Jennifer Listgarten,  Radford M. Neal, Sam T. Roweis, Rachel Puckrin and Sean Cutler,
						<BR>
						         <EM>NIPS</EM> &nbsp; 2006
						<BR>
						<EM>Best Student Paper, Honorable Mention.</EM>
					        (<A HREF = "http://www.cs.toronto.edu/~jenn/HBCPM/">abstract, paper</A>)
		</p></article>

	   <article><p>
	   <H3>
					 Statistical and computational methods for comparative proteomic profiling using
					liquid chromatography-tandem mass spectrometry.
										</H3>
				        Jennifer Listgarten and Andrew Emili,<BR>
					<EM>Molecular and Cellular Proteomics</EM>
					&nbsp; 2005 4:419-434.
					<A HREF = "http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15741312&itool=iconabstr"> (abstract)</A>
					    <A HREF =  "./papers//Listgarten_Emili_MCP_2005.pdf"> (paper)</A>
		</p></article>

		<article><p>
			<H3>
				  Multiple alignment of continuous time series.
				  					</H3>
				Jennifer Listgarten,  Radford M. Neal, Sam T. Roweis and Andrew Emili,<BR>
				            <EM>NIPS</EM> &nbsp; 2004 <BR>
				             <A HREF = "http://www.cs.toronto.edu/~jenn/alignmentStudy">
					      (abstract, paper, slides, and audio demo)</A><BR>
		</p></article>



			</div>
			<div class="clear"></div>
		</section>

<!------------------------------------->
	<section>
			<div class="sectionTitle">
				<h1>All Publications</h1>
			</div>

			<div class="sectionContent">

			<p style="margin-left: 40px">

	<article><P>
	 <H3>Guide your favorite protein sequence generative model</H3>
	Junhao Xiong*, Hunter Nisonoff*, Maria Lukarska*, Ishan Gaur, Luke M. Oltrogge, David F. Savage+, Jennifer Listgarten+
	 <BR>
	  <EM> arXiv</EM> 2025
	 <A HREF="https://arxiv.org/abs/2505.04823">(link</A>)
	 <BR>
		*(equal contributions), +(co-corresponding)
   </P></article>

	<article><P>
	 <H3>Unlocking Guidance for Discrete State-Space Diffusion and Flow Models</H3>
	{Hunter Nisonoff*, Junhao Xiong*, Stephan Allenspach*}, Jennifer Listgarten
	 <BR>
	  <EM> ICLR</EM> 2025
	 <A HREF="https://iclr.cc/virtual/2025/poster/29286">(link</A>)
		*(equal contributions)
	 <BR>
	</P></article>

	<article><P>
		 <H3>Learning antibody sequence constraints from allelic inclusion</H3>
		Milind Jagota, Chloe Hsu, Thomas Mazumder, Kevin Sung, William S DeWitt, Jennifer Listgarten, Frederick A Matsen IV, Chun Jimmie Ye, Yun S Song
		 <BR>
		  <EM> bioRxiv</EM>
		 <A HREF="https://www.biorxiv.org/content/10.1101/2024.10.22.619760v1">(link</A>)
		 2024
			*(equal contributions)
		 <BR>
   </P></article>


   <article><P>
		 <H3>Computationally guided AAV engineering for enhanced gene delivery</H3>
		Jingxuan Guo, Li F Lin, Sydney V Oraskovich, Julio A Rivera de Jesús, Jennifer Listgarten, David V Schaffer
		 <BR>
		  <EM> Trends in Biochemical Sciences</EM> 2024
		 <A HREF="https://www.cell.com/trends/biochemical-sciences/fulltext/S0968-0004(24)00054-9">(link</A>)
		 <BR>
   </P></article>


   <article><P>
		 <H3>GENTANGLE: integrated computational design of gene entanglements</H3>
		Jose Manuel Martí, Chloe Hsu, Charlotte Rochereau, Chenling Xu, Tomasz Blazejewski, Hunter Nisonoff, Sean P Leonard, Christina S Kang-Yun, Jennifer Chlebek, Dante P Ricci, Dan Park, Harris Wang, Jennifer Listgarten, Yongqin Jiao, Jonathan E Allen
		 <BR>
		  <EM> Bioinformatics</EM> 2024
		 <A HREF="https://academic.oup.com/bioinformatics/article-abstract/40/7/btae380/7697098">(link</A>)
		 <BR>
   </P></article>


   <article><P>
		 <H3>Generative models for protein structures and sequences</H3>
		Chloe Hsu, Clara Fannjiang, Jennifer Listgarten
		 <BR>
		  <EM> Nature Biotechnology</EM> 2024
		 <A HREF="https://www.nature.com/articles/s41587-023-02115-w">(link</A>)
		 and a freely available copy <A HREF="./papers/s41587-023-02115-w.pdf">here</A>.
		 <BR>
   </P></article>

   	<article><P>
		<H3>The perpetual motion machine of AI-generated data and the distraction of "ChatGPT as scientist"</H3>
		Jennifer Listgarten
		 <BR>
			  <EM> Nature Biotechnology</EM> 2024
			 <A HREF="https://www.nature.com/articles/s41587-023-02103-0?">(link</A>)
			 					 and a <A HREF="https://rdcu.be/dwSSy"> freely available version here</A>,
			 					 and a <A HREF="https://youtu.be/G43Em6ELaiE?si=Z7NTphG2C1yQS2Js&t=4130"> talk </A>
					 at the 2024 National Academy of Sciences annual meeting.
			 <BR>
   	 </P></article>


	 		   <article><P>
	 				 <H3>Optimal trade-off control in machine learning-based library design, with application to adeno-associated virus (AAV) for gene therapy</H3>
	 				D Zhu*, DH Brookes*, A Busia*, A Carneiro, C Fannjiang, G Popova, D Shin, KC Donohue,, LF Lin, ZM Miller, ER Williams, EF Chang, TJ Nowakowski, <U>J Listgarten*</U>,  <U>DV Schaffer*</U>
	 				(*equal contributions, <U>corresponding</U>)
	 				 <BR>
	 				  <EM>Science Advances</EM> 2024
	 				 (<A HREF="https://www.science.org/doi/10.1126/sciadv.adj3786">paper link</A>)
	 				 <BR>
	 		</p></article>


	 			<article><P>
	 				 <H3>Effective training of nanopore callers for epigenetic marks with limited labelled data</H3>
	 				Brian Yao, Chloe Hsu, Gal Goldner, Yael Michaeli, Yuval Ebenstein, Jennifer Listgarten
	 				 <BR>
	 				  <EM>Open Biology</EM> 2024, 								 &nbsp;
	 				 <A HREF="https://royalsocietypublishing.org/doi/10.1098/rsob.230449">(paper link</A>)
	 				 <BR>
	 			</p></article>


			   <article><P>
					 <H3>Is novelty predictable?</H3>
					Clara Fannjiang and Jennifer Listgarten
					 <BR>
				  <EM>Cold Spring Harbor Perspectives in Biology</EM> 2023
									 (<A HREF="https://doi.org/10.1101/cshperspect.a041469">paper link</A>),
					  similar version also available on <A HREF="https://arxiv.org/abs/2306.00872"> arXiv</A>).
					 <BR>
				</P></article>

			 <article><p>
				 <H3>MBE: model-based enrichment estimation and prediction for differential sequencing data</H3>
				Akosua Busia and Jennifer Listgarten
				 <BR>
				  <EM>Genome Research</EM> 2023, 								 &nbsp;
				 <A HREF="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03058-w">(paper link</A>)
				 <BR>
			</p></article>

		   <article><P>
					 <H3>Coherent Blending of Biophysics-Based Knowledge with Bayesian Neural Networks for Robust Protein Property Prediction</H3>
					Hunter Nisonoff, Yixin Wang, Jennifer Listgarten
					 <BR>
					  <EM>ACS Synth. Biol.</EM> 2023 (Editor's Choice) &nbsp;
					 <A HREF="https://pubs.acs.org/doi/full/10.1021/acssynbio.3c00217?ref=vi_aiforsynbio">(paper link</A>)
					 <BR>
			</P></article>


		   <article><P>
				 <H3>Conformal Prediction under Feedback Covariate Shift for Biomolecular Design</H3>
				 Clara Fannjiang, Stephen Bates, Anastasios Angelopoulos, Jennifer Listgarten, and Michael I. Jordan
				 <BR>
				  <EM>PNAS</EM> 2022, 								 &nbsp;
				 <A HREF="https://www.pnas.org/doi/10.1073/pnas.2204569119">(paper link</A>)
				 <BR>
			</P></article>

			   <article>
					 <H3>On the sparsity of fitness functions and implications for learning</H3>
					David H Brookes, Amirali Aghazadeh, Jennifer Listgarten
					 <BR>
					  <EM>PNAS</EM> 2022, 								 &nbsp;
					 <A HREF="https://www.pnas.org/content/119/1/e2109649118">(paper link</A>)
					 <BR>
				</p></article>

 				<article>
						 <H3>Learning protein fitness models from evolutionary and assay-labeled data</H3>
						Chloe Hsu,  Hunter Nisonoff,  Clara Fannjiang, Jennifer Listgarten
						 <BR>
						  <EM>Nature Biotechnology</EM> 2022								 &nbsp;
						 <A HREF="https://www.nature.com/articles/s41587-021-01146-5">(paper</A>)
						 <BR>
				</p></article>

				<article>
					 <H3>Epistatic Net allows the sparse spectral regularization of deep neural networks for inferring fitness functions</H3>
					Amirali Aghazadeh,  Hunter Nisonoff,  Ocal, Yijie Huang, O. Ozan Koyluoglu,  Jennifer Listgarten, Kannan Ramchandran
					 <BR>
					  <EM>Nature Communications</EM> 2021, 								 &nbsp;
					 <A HREF="https://www.nature.com/articles/s41467-021-25371-3">(paper link</A>)
					 <BR>
				</p></article>

				<article><P>
						 <H3>Autofocused oracles for model-based design</H3>
						 Clara Fannjiang and Jennifer Listgarten
						<BR>
						<EM>NeurIPS</EM> 2020, 								 &nbsp;
						<A HREF="https://arxiv.org/abs/2006.08052">(paper link</A>)
						 <BR>											</p></article>

				<article><p>
														 <H3>Rethinking drug design in the artificial intelligence era</H3>

														 P Schneider, WP Walters, AT Plowright, N Sieroka,
														 J Listgarten, RA Goodnow Jr., J Fisher,
														 JM Jansen, JS Duca, TS Rush, M Zentgraf,
														 JE Hill, E Krutoholow, M Kohler, J Blaney,
														 K Funatsu, C Luebkemann and G Schneider
														 <BR>
														 in <EM>Nature Reviews Drug Discovery</EM>
														 &nbsp;2019
														 <A HREF="https://www.nature.com/articles/s41573-019-0050-3?draft=collection">(paper link)</A>
														<BR>
				</p></article>

			<article><p>
							 <H3>A view of Estimation of Distribution Algorithms through the lens of Expectation-Maximization</H3>

							 David H. Brookes, Akosua Busia, Clara Fannjiang, Kevin Murphy and Jennifer Listgarten

							 <BR>
							 in <EM>Proceedings of GECCO 2020</EM>
							 &nbsp; (<A HREF="https://dl.acm.org/doi/10.1145/3377929.3389938">here</A>
							 , extended version
							 <A HREF="https://arxiv.org/abs/1905.10474">here</A>)
						    <BR>
			</p></article>

			<article><p>
					 <H3>Conditioning by adaptive sampling for robust design</H3>
					 David H. Brookes, Hahnbeom Park and Jennifer Listgarten

					 <BR>
					 accepted at <EM>ICML</EM>
					 &nbsp;2019
					  <A HREF="https://arxiv.org/abs/1901.10060">(paper link</A>, <I>arXiv</I> version is most up-to-date)
					 <BR>
					 (5% acceptance rate for 20 min. oral presentation)
					<BR>

				</p></article>

			<article><p>
													<H3>
													 Design by adaptive sampling
														</H3>
													 David Brookes and Jennifer Listgarten
													 <BR>
													 <EM>NeurIPS Workshop on Machine Learning for Molecules and Materials</EM>
													 &nbsp;2018
													 <A HREF="https://arxiv.org/abs/1810.03714">(paper link)</A>
													 <BR>
			</p></article>

										<article><p>
											<H3>
											 Gaussian Process Prior Variational Autoencoders
												</H3>
											 Francesco Paolo Casale, Adrian V Dalca, Luca Saglietti,
	 										  Jennifer Listgarten, Nicolo Fusi
											 <BR>
											 in <EM>NeurIPS</EM>
											 &nbsp;2018
											 <A href="https://papers.nips.cc/paper/2018/hash/1c336b8080f82bcc2cd2499b4c57261d-Abstract.html">(paper link)</A>

										</p></article>


							<article><p>


						<article><p>
								<H3>
								 Predicting off-target effects for end-to-end CRISPR guide design
									</H3>
								 <U>J Listgarten</U>, <U>M Weinstein</U>, B Kleinstiver, AA Sousa, JK Joung, J Crawford, K Gao, M Elibol, L Hoang, <U>J Doench</U>, <U>N Fusi</U>
								 <U>(equal contributions and corresponding</U>)
								 <BR>
								 <EM>Nature Biomedical Engineering</EM>
								 &nbsp;(2018)
								 <A HREF="https://www.nature.com/articles/s41551-017-0178-6">(paper link)</A>
								 <BR>
								 <FONT color="#000099">
								  Associated tools and resources available
								  <A HREF="https://www.microsoft.com/en-us/research/project/crispr/">here</A>.
								 </FONT>
							</p></article>


				<article><p>
								<H3>
								 Orthologous CRISPR-Cas9 for Combinatorial Genetic Screens
									</H3>
								 F Najm*, C Strand*, K Donovan*, M Hegde*, KR. Sanson*, EW Vaimberg, ME Sullender, E Hartenian, N Fusi,
								 J. Listgarten, ST Younger*, BE Bernstein**, DE Root**, JG Doench**
								 <BR>
								 <EM>Nature Biotechnology</EM>
								 &nbsp;(2018)
								 <A HREF="https://www.nature.com/articles/nbt.4048">(paper link)</A>
								 (*equal contributions, **corresponding)
								 <BR>
				</p></article>


  				<article><p>
								<H3>
								Identifying gene expression modules that define human cell fates
								</H3>
								<U>I Germanguz</U>, <U>J Listgarten</U>, <U>A Solomon</U>, X Gaeta, WE Lowry
								&nbsp;(<U>equal contributions</U>)
								<BR>
								<EM>
								Stem Cell Research
								</EM>
								&nbsp;(2016,, in press)
				</p></article>



			    <article><p>
								<H3>
								Leveraging Non-Linear Genetic Effects on Functional Traits for GWAS
								</H3>
								Nicolo Fusi and Jennifer Listgarten
								<BR>
								<EM>RECOMB Proceedings</EM> (in <EM>Lecture Notes in Computer Science</EM>)
								</EM>
								&nbsp;2016

								(<A HREF="./papers/2016.FusiListgarten.fGwas.RECOMB.pdf">pdf</A>)
				</p></article>

				<article>
										<p>
																		<h3>
																		Optimized sgRNA design to maximize activity and minimize off-target effects for genetic screens with CRISPR-Cas9
																		<!--
																		<img src="images/new.gif" >
																		-->
																		</h3>
																		<U>JG Doench</U>*, <U>N Fusi</U>*, M Sullender*, M Hegde*, EW	Vaimberg*, KF Donovan, I Smith, Z Tothova,
																		C Wilen , R Orchard , HW Virgin, <U>J Listgarten</U>*, <U>DE Root</U>
																		<BR>
																		<EM>Nature Biotechnology</EM> Jan 2016
																		<A HREF="http://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.3437.html">doi:10.1038/nbt.3437</A>
																		<BR>
																		(*equal contributions, <U>corresponding</U>)
																		<BR>
																		<FONT color="#000099">
																		A pre-print of just the computational aspects of this paper is available
																		on <EM> <A HREF="http://biorxiv.org/content/early/2015/06/26/021568">
																				bioRxiv</A></EM>

																		<BR>
																		Source code and prediction server available from here:
																		<A HREF="
																		http://research.microsoft.com/en-us/projects/azimuth
																		">here</A>.
																		<BR>
																		[<A HREF="http://blogs.microsoft.com/next/2016/01/18/molecular-biology-meets-computer-science-tools-in-new-system-for-crispr/">Microsoft Research blog post</A>]
																		<BR>
																		[<A HREF="https://www.broadinstitute.org/blog/machine-learning-approach-improves-crispr-cas9-guide-pairing">Broad Institute blog post</A>]

																		</FONT>
								</p>
				</article>



				<article><p>
				<H3>
				In Silico Predictive Modeling of CRISPR/Cas9 guide efficiency
				</H3>
				Nicolo Fusi, Ian Smith, John Doench, Jennifer Listgarten
				<BR>
				<EM>
				bioRxiv, dx.doi.org/10.1101/021568
				</EM>
				2015 (
				<A HREF="http://biorxiv.org/content/early/2015/06/26/021568">
				preprint
				</A>
				)
				<BR>
				<FONT color="#000099">
				This pre-print has been largely (though not entirely) absorbed into the
				Nature Biotechnology paper above.
				</FONT>
				</p></article>


				<article><p>
				<H3>
				Further Improvements to Linear Mixed Models for Genome-Wide Association Studies
				</H3>
				Chris Widmer, Christoph. Lippert, Omer Weissbrod, Nicolo Fusi, Carl Kadie, Bob Davidson, Jennifer Listgarten and D. Heckerman
				<BR>
				<EM>
				Scientific Reports,
				</EM>
				Nov. 2014
				<A HREF="
				http://www.nature.com/srep/2014/141112/srep06874/full/srep06874.html
				">
				(open access)
				</A>
				</p></article>


				<article><p>
				<H3>
				let-7 miRNAs Can Act through Notch to Regulate Human Gliogenesis
				</H3>
				Patterson M, Gaeta X, Loo K, Edwards M, Smale S, Cinkornpumin J, Xie Y, Listgarten J, Azghadi S, Douglass SM, Pellegrini M, Lowry WE.
				<BR>
				<EM>
				Stem Cell Reports
				</EM>
				2014,
				doi: 10.1016/j.stemcr.2014.08.015
				<A HREF="
				http://www.cell.com/stem-cell-reports/pdfExtended/S2213-6711(14)00269-0
				">
				(open access)
				</A>
				</p></article>


				<article><p>
				<H3>
				Personalized Medicine: From Genotypes, Molecular Phenotypes and the Quantified Self, Toward Improved Medicine
				</H3>
				Joel Dudley, Jennifer Listgarten, Oliver Stegle, Steven Brenner, Leopold Parts
				<BR>
				<EM>
				Proceedings of the Pacific Symposium on Biocomputing
				</EM>
				2015
				<A HREF="
				http://psb.stanford.edu/psb-online/proceedings/psb15/intro-pm.pdf
				">(pdf)
				</A>
				</p></article>

				<article><p>
				<H3>
				Greater power and computational efficiency for kernel-based association testing of sets of genetic variants
				</H3>
				C. Lippert, J. Xiang, D. Horta, C. Widmer, C. Kadie, D. Heckerman, Jennifer Listgarten
				<BR>
				<EM>
				Bioinformatics
				</EM>
				2014,
				doi: 10.1093/bioinformatics/btu504
				<A HREF="
				http://bioinformatics.oxfordjournals.org/content/early/2014/09/07/bioinformatics.btu504
				">
				(open access)
				</A>
				</p></article>


				 <article><p>
				 <H3>
				 Epigenome-wide association studies without the need for cell-type composition
				 </H3>
				 James Zou, Christoph Lippert, David Heckerman, Martin Aryee, Jennifer Listgarten
				 <BR>
				 <EM>Nature Methods</EM>, 309-311 (2014)
				 <A HREF="http://www.nature.com/nmeth/journal/v11/n3/full/nmeth.2815.html"> (journal link)</A>
				 <BR>
				 <FONT color="#000099">
				  Python software available from <A HREF="http://research.microsoft.com/en-us/downloads/8af2ab12-a205-4bbb-809c-a333ecaffa40/">here</A>, and
				 R software available from <A HREF="http://research.microsoft.com/en-us/downloads/472fe637-7cb9-47d4-a0df-37118760ccd1/">here</A>.
				 </FONT>
				 <BR> Corrigendum: Supp. Figure 1 was not run with filters as described in the paper, but without any filters. We have contacted the journal to post this correction.
				 </p></article>


				<article><p>
				<H3>
				Personalized Medicine: from genotypes and molecular phenotypes toward therapy
				</H3>
				Jennifer Listgarten, Oliver Stegle, Quaid Morris, Steven Brenner,  Leo Parts
				<BR>
				<EM>
				Proceedings of the Pacific Symposium on Biocomputing
				</EM>
				2014
				</p></article>


				<article><p>
				<H3>
				A genome-to-genome analysis of associations between human genetic variation, HIV-1 sequence diversity, and viral control.
				</H3>
				I. Bartha, J. Carlson, C. Brumme, P. McLaren, Z. Brumme, M. John, D. Haas, J. Martinez-Picado, J. Dalmau, C. LÃ³pez-GalÃ­ndez, C. Casado, A. Rauch, H. GÃ¼nthard, E. Bernasconi, P. Vernazza, T. Klimkait, S. Yerly, S. OâBrien, Jennifer Listgarten, N. Pfeifer, C. Lippert, N. Fusi, Z. Kutalik, T. Allen, Viktor MÃ¼ller, R. Harrigan, D. Heckerman, A. Telenti, J. Fellay
				<BR>
				<EM>
				eLife
				</EM>
				(2013) 2:e01123
				(<A HREF="
				http://elife.elifesciences.org/content/2/e01123
				">journal link</A>)
				<FONT color="#000099">
				</FONT>
				</p></article>


				<article><p>
				<H3>
				The benefits of selecting phenotype-specific variants for applications of mixed models in genomics.
				</H3>
				<U>C. Lippert</U>,
				G. Quon, EY Kang, C. Kadie,
				<U>Jennifer Listgarten</U>,
				<U>D. Heckerman</U>
				<BR>
				(<U>equal contributions</U>)
				<BR>
				<EM>
				Scientific Reports
				</EM>
				 (2013) doi:10.1038/srep01815
				(<A HREF="
				http://www.nature.com/srep/2013/130509/srep01815/full/srep01815.html
				">journal link</A>)
				<FONT color="#000099">
				</FONT>
				</p></article>

				<article><p>
				<H3>
				FaST-LMM-Select for addressing confounding from spatial structure and rare variants
				</H3>
				<U>Jennifer Listgarten</U>, <U>Christoph Lippert</U>, <U>David Heckerman</U>

				(<U>equal contributions and corresponding</U>)
				<BR>
				<EM>
				Nature Genetics,
				</EM>
				45, 470-471 (2013) doi:10.1038/ng.2620
				(<A HREF="
				http://www.nature.com/ng/journal/v45/n5/full/ng.2620.html
				">journal link</A>)
				</p></article>

				<article><p>
				<H3>
				A powerful and efficient set test for genetic markers that handles confounders
				</H3>
				<U>Jennifer Listgarten</U>, <U>C. Lippert</U>, EY Kang, J. Xiang, C. Kadie, <U>D. Heckerman</U>
				<BR>
				(<U>equal contributions and corresponding</U>)
				<BR>
				<EM>
				Bioinformatics
				</EM>
				2013,
				doi: 10.1093/bioinformatics/btt177
				<A HREF="
				http://bioinformatics.oxfordjournals.org/content/29/12/1526
				">
				(open access)
				</A>
				<BR>
				<FONT color="#000099">
				Source and executables available
				<A HREF="
				http://www.jennifer.listgarten.com/fastlmm.html
				">here</A>.
				</FONT>
				</p></article>


				<article><p>
				<H3>
				An Exhaustive Epistatic SNP Association Analysis on Expanded Wellcome Trust Data
				</H3>
				<U>C. Lippert</U>,
				<U>Jennifer Listgarten</U>,
				<U>R. Davidson</U>,
				S. Baxter,
				H. Poon,
				C. Kadie,
				<U>D. Heckerman</U>,
				<BR>
				<U>(equal contributions)</U>
				<BR>
				<EM>
				Scientific Reports,
				</EM>
				2013,
				<A HREF="
				http://www.nature.com/srep/2013/130122/srep01099/full/srep01099.html
				">doi:10.1038/srep01099
				</A>
				</p></article>

				<article><p>
				<H3>
				Patterns of methylation heritability in a genome-wide analysis of four brain regions
				</H3>
				Gerald Quon,
				Christoph Lippert,
				David Heckerman,
				Jennifer Listgarten
				<BR>
				<EM>
				Nucleic Acids Research,
				</EM>
				2013,
				<A HREF="
				http://nar.oxfordjournals.org/content/41/4/2095
				">doi: 10.1093/nar/gks1449
				</A>
				<!----
				<BR>
				<FONT color="#000099">
							    Matlab and R data files containing our narrow-sense heritability estimates, p-values, as
								well as BED files of our heritable sites and downloaded chromatin annotations
								<A HREF="http://people.csail.mit.edu/geraldquon/MSR/final50kbPvalues.zip"> can be found here</A>.
								---->
				</FONT>
				<BR>
		</p></article>

				<article><p>
				<H3>
				The future of genome-based medicine.
				</H3>
				Quaid Morris, Steven Brenner, Jennifer Listgarten, Oliver Stegle
				<BR>
				<EM>
				Proceedings of the Pacific Symposium on Biocomputing
				</EM>
				2013, 16:456-458. doi:10.1142/9789814447973_0046
				</p></article>




				<article><p>
				    <H3> Correlates of Protective Cellular Immunity Revealed by Analysis
				of Population-Level Immune Escape Pathways in HIV-1 </H3>

				J. Carlson,
				C. Brumme,
				E. Martin,
				Jennifer Listgarten,
				M. Brockman,
				AQ. Le,
				C. Chui,
				L. Cotton,
				D. Knapp,
				SA. Riddler,
				R. Haubrich,
				G. Nelson,
				N. Pfeifer,
				C. DeZiel,
				D. Heckerman,
				R. Apps,
				M. Carrington,
				S. Mallal,
				R. Harrigan,
				M. John,
				Z. Brumme
				and the International HIV Adaptation Collaborative
				<BR> <EM> Journal of Virology, </EM> Dec. 2012,
				<A HREF="
				http://jvi.asm.org/content/86/24/13202
				">86(4)</A> <BR> <H3> <FONT color="#000099"> </FONT></H3>
		</p></article>

				<article><p>
				<H3>
				Co-Operative Additive Effects between HLA Alleles in Control of HIV-1
				</H3>
				<U>P. Matthews</U>, <U>Jennifer Listgarten</U>, J.
				Carlson, R. Payne, KH Huang, J Frater,
				D Goedhals, D Steyn, D van Vuuren, P Paioni,
				P Jooste, A Ogwu, R Shapiro, Z Mncube, T
				Ndung'u, B Walker, D Heckerman, P
				Goulder
				<BR>
				(<U>equal contributions</U>)
				<BR>
				<EM>
				PLoS One,
				</EM>
				2012,
				<A HREF="
				http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0047799
				">7(10): e47799. doi:10.1371/journal.pone.0047799
				</A>
				<BR>
		</p></article>

				<article><p>
				<H3>
				Improved linear mixed models for genome-wide association studies
				</H3>
				<U>Jennifer Listgarten</U>, <U>C Lippert</U>, CM Kadie, RI Davidson, E Eskin and
				<U>D Heckerman</U>
				<BR>
				(<U>equal contributions and corresponding</U>)
				<BR>
				<EM>
				Nature Methods,
				</EM>
				2012,
				<A HREF="
				http://www.nature.com/nmeth/journal/v9/n6/abs/nmeth.2037.html
				">doi:10.1038/nmeth.2037
				</A>
				<BR>
				<FONT color="#000099">
				Source and executables available
				<A HREF="
				http://www.jennifer.listgarten.com/fastlmm.html
				">here</A>.
				</FONT>
				</p></article>


				<article><p>
				<H3>
				Learning Transcriptional Regulatory Relationships Using Sparse Graphical Models
				</H3>
				X Zhang, W Cheng, Jennifer Listgarten, C Kadie, S Huang, W Wang, D Heckerman
				<BR>
				<EM>
				PLoS One,
				</EM>
				2012,
				<A HREF="
				http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0035762
				">
				doi:10.1371/journal.pone.0035762
				</A>
				</p></article>


				<article><p>
				<H3>
				Widespread Impact of HLA Restriction on Immune Control and Escape Pathways in HIV-1
				</H3>
				J. Carlson, Jennifer Listgarten, N Pfeifer, V Tan, Carl Kadie, B Walker, T Ndung'u, R Shapiro, J Frater, Z Brumme, P Goulder and D Heckerman
				<BR>
				<EM>
				Journal of Virology,
				</EM>
				February 2012,
				doi:10.1128/?JVI.06728-11
				(<A HREF="
				http://jvi.asm.org/content/early/2012/02/22/JVI.06728-11.abstract
				">abstract</A>,<A HREF="
				http://jvi.asm.org/content/early/2012/02/22/JVI.06728-11.full.pdf+html
				">paper</A>)
				</p></article>

				<article><p>
				<H3>
				Personalized Medicine: From Genotype and Molecular Phenotypes Towards Computed Therapy
				</H3>
				Oliver Stegle, Frederick P. Roth, Quaid Morris, Jennifer Listgarten
				<BR>
				<EM>
				Proceedings of the Pacific Symposium on Biocomputing
				</EM>
				2012
				</p></article>

				<article><p>
				<H3>
				FaST Linear Mixed Models for Genome-Wide Association Studies
				</H3>
				<U>C Lippert</U>, <U>Jennifer Listgarten</U>, Y Liu, CM Kadie, RI Davidson and <U>D Heckerman</U>
				<BR>
				(<U>equal contributions and corresponding</U>)
				<BR>
				<EM>
				Nature Methods,
				</EM>
				Aug. 2011,
				<A HREF="
				https://www.nature.com/articles/nmeth.1681
				">doi:10.1038/nmeth.1681
				</A>,
				<BR>
				<A HREF="
				./papers/FaST-LMM-2011.pdf">
				freely available author copy including SI</A>,
				<FONT color="#000099">
				Source and executables available
				<A HREF="
				http://www.jennifer.listgarten.com/fastlmm.html
				">here</A>.
				</FONT>
				</p></article>


				<article><p>
				<H3>
				HLA-A*7401-mediated control of HIV viremia is independent of its linkage
				disequilibrium with HLA-B*5703.
				</H3>
				P. Matthews, E. Adland, J. Listgarten, A. Leslie, N. Mkhwanazi, J. Carlson, M. Harndahl, A. Stryhn, R.
				Payne, A. Ogwu, K. Huang, J. Frater, P. Paioni, H. Kloverpris, P.Jooste, D. Goedhals, C. van Vuuren, D.
				Steyn, L. Riddell, F. Chen, G. Luzzi, T. Balachandran, T. Ndung'u, S. Buus, M. Carrington, R. Shapiro, D.
				Heckerman, and P. Goulder
				<BR>
				<EM>
				Journal of Immunology
				</EM>April 2011,
				<A HREF="
				http://www.jimmunol.org/content/early/2011/04/15/jimmunol.1003711.abstract
				">doi: 10.4049
				</A>
				</p></article>



				<article><p>
				<H3>
				Correction for Hidden Confounders in the Genetic Analysis of Gene Expression
				</H3>
				Jennifer Listgarten, Carl Kadie, Eric Schadt, David Heckerman
				<BR>
				<EM>
				Proceedings of the National Academy of Sciences,
				</EM>September 1, 2010, <A HREF="http://www.pnas.org/content/early/2010/08/30/1002425107.abstract">doi: 10.1073/pnas.1002425107
				</A>
				<BR>
				(<A
				HREF="http://www.pnas.org/content/early/2010/08/30/1002425107.full.pdf+html
				">paper</A>,
				<A
				HREF="http://www.pnas.org/content/suppl/2010/09/01/1002425107.DCSupplemental/Appendix.pdf
				">Supplementary Information</A>,
				<A
				HREF="http://mscompbio.codeplex.com/wikipage?title=eLMM&referringTitle=Home
				">software</A>,
				<A
				HREF="http://www.genomeweb.com/informatics/microsoft-research-led-team-develops-method-correct-confounders-eqtl-analyses
				">GenomeWeb coverage</A>)
				</p></article>


				<article><p>
				<H3>
				Additive contribution of HLA class I alleles in the immune control of HIV-1 infection
				</H3>
				Leslie A, Matthews PC, Listgarten J, Carlson JM, Kadie C, Ndung'u T, Brander C, Coovadia H, Walker BD,
				Heckerman D, Goulder PJ
				<BR>
				<EM>
				Journal of Virology
				</EM>, 2010
				</p></article>


				<article><p>
				<H3>
				Rare HLA Drive Additional HIV Evolution Compared to More Frequent Alleles
				</H3>
				CM Rousseau, DW Lockhart, Jennifer Listgarten, C Kadie, GH
				Learn, DC Nickle, D Heckerman, W Deng, C Brander, T
				Ndung'u, H Coovadia, P Goulder, B. Korber, B Walker, J Mullins
				<BR>
				<EM>
				AIDS Research and Human Retroviruses
				</EM>, 2009; 25(3):297-303
				</p></article>

				<article><p>
				<H3>
				<EM>In silico</EM> resolution of ambiguous HLA typing data
				</H3>
				J Listgarten, Z Brumme, C Kadie, G Xiaojiang, B Walker, M
				Carrington, P Goulder, D Heckerman,
				<BR>
				in <EM> ASHI  Quarterly, Volume 32, Number 2, 2008</EM>
				<BR>

				<FONT color="#000099">
				For the public web server tool based on this work,
				go
				<A HREF="
				https://phylod.research.microsoft.com/Tools/SelectToolParams?toolId=9
				"> here </A>;
				for .exe and source code (training code not included), go
				<A HREF="
				http://www.codeplex.com/MSCompBio/Wiki/View.aspx?title=Hla%20Completion
				"> here</A>.
				</FONT>
				(<A
				HREF="./papers/ASHIquareterlyHLARefinement08.pdf
				">pdf</A>)


				</p></article>

	<article><p>
				<H3>
				Statistical resolution of ambiguous HLA typing data.
				</H3>
				Jennifer Listgarten, Z Brumme, C Kadie, G Xiaojiang, B Walker, M
				Carrington, P Goulder, D Heckerman,
				<BR>
				in <EM> PLoS Computational Biology</EM>, 2008, 4(2):e1000016
				<BR>
				<FONT color="#000099">
				For the public web server tool based on this work,
				go
				<A HREF="
				https://phylod.research.microsoft.com/Tools/SelectToolParams?toolId=9
				"> here </A>;
				for .exe and source code (training code not included), go
				<A HREF="
				http://www.codeplex.com/MSCompBio/Wiki/View.aspx?title=Hla%20Completion
				"> here</A>.
				</FONT>

				<BR>
								(<A
								HREF="
								http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1000016
								">abstract</A>,
								<A
								HREF="
								./papers/resolutionHLAPLoSCompBio2008.pdf
								">paper</A>,
								<A HREF="./papers/BioInformApril18.2008.pdf">coverage in the magazine BioInform</A>,
								<A HREF="http://www.eurekalert.org/pub_releases/2008-02/plos-itq022208.php">press release</A>)

				</p></article>


				<article><p>
				<H3>
				A statistical framework for modeling HLA-dependent T-cell response data.</H3>


				Jennifer Listgarten, Nicole Frahm, Carl Kadie, Christian Brander and David
				Heckerman,
				<BR>
				<EM> PLoS Computational Biology</EM>, 2007, 3(10):e188
				<BR>
				<FONT color="#000099">
				Web tool, executable and source code available
				<A HREF="
				http://atom.research.microsoft.com/bio/
				">
				here, under "HLA Assignment"
				</A>
				</FONT>
				<BR>
				(<A
				HREF="http://compbiol.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pcbi.0030188
				">abstract, paper</A>,
				<A HREF="
				http://www.eurekalert.org/pub_releases/2007-10/plos-mpr101007.php
				">press release</A>)

				</p></article>

				<article><p>
				<H3>
				Extensive HLA class I allele promiscuity among viral CTL epitopes.
				</H3>

				N. Frahm, K. Yusim, T. Suscovich, S. Adams, J. Sidney, P. Hraber, H. Hewitt, CH. Linde, D. Kavanagh, T.
				Woodberry, L. Henry, K. Faircloth, J. Listgarten, C. Kadie, N. Jojic, K. Sango, N. Brown, E. Pae, M. Zaman, F.
				Bihl, A. Khatri, M. John, S. Mallal, F. Marincola, B. Walker, A. Sette, D. Heckerman, B. Korber, C. Brander
				<BR>
				<EM>European Journal of Immunology</EM>, 2007 37(9):2419-2433.
				<BR>
				<FONT color="#000099">
				See paper above for code/tools used in this paper.
				</FONT>
				(<A HREF=
				"http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=17705138&ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum"
				>abstract</A>)

				</p></article>

				<article><p>
				<H3>
				Evidence that dysregulated DNA mismatch repair characterizes human
				non-melanoma skin cancer
				</H3>
				Leah C. Young, Jennifer Listgarten, Martin J. Trotter, Susan E. Andrew, Victor A. Tron
				<BR>
				<EM>British Journal of Dermatology</EM>, 2008 158(1):59-69.
				(<A HREF="
				http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=17970804&ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum
				">abstract</A>)

				</p></article>


				<article><p>
				<H3>
				Determining the number of non-spurious arcs in a learned DAG model: Investigation of a
				Bayesian and a frequentist approach.
				</H3>

				Jennifer Listgarten and David Heckerman
				<BR>
				<EM>Proceedings of Twenty-Third Conference on Uncertainty in Artificial
				Intelligence</EM>, UAI Press, July 2007
				(<A HREF = "./papers/TR-2007-60.pdf">
				    paper</A>)

				</p></article>

				<article><p>
				      <a name="three"></a>
				<!---
				<img src="images/new.gif" >
				--->
				<H3> Analysis of sibling time series data: alignment and difference
				detection</H3>
				Jennifer Listgarten,<BR>
				Ph.D. Thesis, Department of Computer Science, University of Toronto 2007.
				<BR>
				 (<A HREF = "./papers/phdThesis/index.html">abstract,
				thesis and code</A>)

			</p></article>

	    <article><p>

				<H3>
				<!---
				<img src="images/new.gif" >
				--->
				Bayesian detection of infrequent differences in
				sets of time series with shared structure.
				</H3>
				Jennifer Listgarten,  Radford M. Neal, Sam T. Roweis, Rachel Puckrin and Sean Cutler,
				<BR>
				         <EM>Advances in Neural Information Processing Systems</EM> 19, MIT Press,
				Cambridge, MA, 2007 (
				<A HREF="http://www.nips.cc/Conferences/2006/">
				NIPS 2006</A>).
				<BR>
				<EM>Best Student Paper, Honorable Mention.</EM>
			        (<A HREF = "http://www.cs.toronto.edu/~jenn/HBCPM/">abstract, paper</A>)
		</p></article>

		   <article><p>
				<H3>   Difference detection in LC-MS data for protein biomarker discovery.
				</H3>
				Jennifer Listgarten,  Radford M. Neal, Sam T. Roweis, Peter Wong and Andrew Emili,<BR>
				<EM>Bioinformatics</EM>, 2007 23:e198-e204 (by way of
				<A HREF="http://www.eccb06.org/"> ECCB 2006<A>)
				<BR>
				<EM>Best Student Paper, 3rd prize</EM>
				<BR>
				 <A HREF = "http://www.cs.toronto.edu/~jenn/LCMS">
					      (abstract, paper and data set)</A>
	</p></article>


	<article><p>

				<H3>
				Leveraging information across HLA alleles/supertypes improves epitope prediction.
				</H3>

				     David Heckerman, Carl Kadie, Jennifer Listgarten,<BR>
				<EM>Journal of Computational Biology</EM>, 2007 14: 736-746
				<BR>
				(shorter version also appears
				<EM>Proceedings of Research in Computational Molecular Biology<!---: 10th Annual
				International Conference (RECOMB 2006), Venice, Italy, April 2-5, 2006-->. Lecture
				Notes in Computer Science, Volume 3909, Mar 2006, 296-308</EM>.)
				<BR>
					<A HREF="
								http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=17691891&ordinalpos=2&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum
								">(abstract,</a>
								<A HREF="
				./papers/LeveragingInfoEpitopePredictionJCB2007.pdf">paper)</a>
				<BR>
				<FONT color="#000099">
				Web tool, executable and source code available
				<A HREF="
				http://atom.research.microsoft.com/bio/
				">
				here, under "Epitope Prediction"
				</A>
				</FONT>
				</p></article>
					<article><p>
					<H3> Practical proteomic biomarker discovery: taking a step back to leap
					forward.</H3>
				        Jennifer Listgarten and Andrew Emili,<BR>
				        <EM>Drug Discovery Today</EM>, 2005 10:1697-1702.
				        <BR>
				        <A HREF =  "http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16376831&query_hl=1&itool=pubmed_docsum">
										(abstract)</A>
					<A HREF =  "./papers/ListgartenEmiliDDT2005.pdf"> (paper)</A>
		</p></article>

	<article><p>
					<H3> Statistical and computational methods for comparative proteomic profiling using
					liquid chromatography-tandem mass spectrometry.</H3>
				        Jennifer Listgarten and Andrew Emili,<BR>
					<EM>Molecular and Cellular Proteomics</EM>, 2005 4:419-434.
					<BR>
					 <A HREF = "http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15741312&itool=iconabstr"> (abstract)</A>
					    <A HREF =  "./papers//Listgarten_Emili_MCP_2005.pdf"> (paper)</A>
			</p></article>

		<article><p>
				<H3>  Multiple alignment of continuous time series.</H3>
							Jennifer Listgarten,  Radford M. Neal, Sam T. Roweis and Andrew Emili,<BR>
				            <EM>Advances in Neural Information Processing Systems</EM> 17, MIT Press, Cambridge, MA,
				2005 (NIPS 2004).
				<BR>
				<FONT color="#000099">

				The Continuous Profile Models (CPM) Matlab Toolbox
				is available
				<A HREF="http://www.cs.toronto.edu/~jenn/CPM/">
				here.
				</A>
				</FONT>
				<BR>
				  <A HREF = "http://www.cs.toronto.edu/~jenn/alignmentStudy">
					      (abstract, paper, slides, and audio demo)</A>
				</p></article>


				        <article><p>
				<H3> Predictive models for breast cancer susceptibility from multiple,
				            single nucleotide polymorphisms.</H3>
					    <A HREF = "http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15102677&query_hl=2"> (abstract)</A>
					    <A HREF = "./papers/PredictiveModelsSNPListgarten2005.pdf"> (paper)</A><BR>
				Jennifer Listgarten,  S Damaraju,
				            B Poulin, L Cook, J Dufour, A Driga, J Mackey, D Wishart,
				            R Greiner and B Zanke,<BR>
					<EM>Clinical Cancer Research</EM> 2004:10(8):2725-37.<BR>
				</p></article>

				        <article><p>
				<H3> Clinically validated
				             benchmarking of normalization techniques for two-colour oligonucleotide
				spotted microarray slides.</H3>
					     <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15130793&query_hl=2">
				(abstract)</A>
					     <A HREF="normalizationStudy/paper.html"> (paper)</A><BR>
				Jennifer Listgarten, K Graham,
				             S Damaraju, C Cass, J Mackey and B Zanke,
				             </H3>
				 <EM>Applied Bioinformatics</EM> 2003:2(4)219-228.<BR>
				</p></article>

				        <article><p>
				             <H3> Lymphovascular invasion is associated with poor survival in gastric
				cancer: an application of gene-expression and tissue array techniques.</H3>
				 BJ Dicken, K Graham, SM Hamilton, S Andrews,
				R Lai, Jennifer Listgarten, GS Jhangri, LD Saunders,
				S Damaraju and CE Cass, <BR>
				<EM>Annals of Surgery</EM> 2006: 243(1):64-73.<BR>

				</p></article>

				          <article><p>

				<H3>Exploring qualitative probabilities for image
				          understanding</H3>
				      Jennifer Listgarten,<BR>
				M.Sc. Thesis, Department of Computer Science, University of Toronto, October 2000.
				<BR>
				 <a href="http://www.cs.toronto.edu/~jenn/msc/msc_thesis.pdf"> (pdf 1.2MB)</a>
				          <a href="http://www.cs.toronto.edu/~jenn/msc/msc_thesis.ps.gz"> (ps.gz 0.6MB)</a>
				</p></article>



<!------------------------------------->
		</div>
		<div class="clear"></div>
	</section>



	</div>
</div>


<!--footer code begin-->
<!---
<div style="font-family: Verdana; width:1002px; height:25px; padding-top:5px; font-size:60%;">
<a href="/c/1450" style="border-right:1px solid #fff; padding:0em 1em 0em 0em;">Contact</a>
<a href="/c/1452" style="border-right:1px solid #fff; padding:0em 1em 0em 1em;">Terms</a>
<a href="/c/1453" style="border-right:1px solid #fff; padding:0em 1em 0em 1em;">Trademarks</a>
<a href="/c/1451" style="border-right:1px solid #fff; padding:0em 1em 0em 1em;">Privacy and Cookies</a>
<a href="/c/1454" style="padding:0em 1em 0em 1em;">Code of Conduct</a>
<span style="margin-right:100px;">&copy;<span id="copyrightYear"></span> Microsoft Corporation. All rights reserved.</span><a href="/c/1456"><img style="margin-right:10px;" alt="Microsoft" src="/a/i/c/logo_ms.png" border="0"></a>
<script language="javascript" src="/a/year.js" type="text/javascript"></script>
</div>
--->
<!--footer code end-->



<!-- Start of StatCounter Code -->
<!--
<script type="text/javascript" language="javascript">
var sc_project=1621305;
var sc_invisible=1;
var sc_partition=15;
var sc_security="d97fa776";
var sc_remove_link=1;
</script>

<script type="text/javascript" language="javascript"
src="http://www.statcounter.com/counter/counter.js"></script><noscript><img
src="http://c16.statcounter.com/counter.php?sc_project=1621305&java=0&security=d97fa776&invisible=1"
alt="free stats" border="0"> </noscript>
<!-- End of StatCounter Code -->
--->


<!-- WiredMinds eMetrics tracking with Enterprise Edition V5.4 START -->
<script type='text/javascript' src='https://count.carrierzone.com/app/count_server/count.js'></script>
<script type='text/javascript'><!--
wm_custnum='d94c48033c50baf8';
wm_page_name='index.html';
wm_group_name='/services/webpages/l/i/listgarten.com/jennifer/public';
wm_campaign_key='campaign_id';
wm_track_alt='';
wiredminds.count();
// -->
</script>
<!-- WiredMinds eMetrics tracking with Enterprise Edition V5.4 END -->
</body>
</html>
